src/utils/scrappers/ectsScrapper/checkSubjects.ts

```diff
@@ -0,0 +1,4 @@
+const checkSubjects = (subject: string) =>
+    subject === 'Informatyka' || subject === 'Matematyka';
```

Comment:
Same as above, maybe move these names to a constant in this file or to a separate file with constants?